_delphi_utils_python/delphi_utils/flash_eval/eval_day.py

```diff
@@ -0,0 +1,332 @@
+"""Functions pertaining to running FlaSH daily."""
+
+import numpy as np
+import pandas as pd
+from scipy.stats import binom
+import boto3
+from delphi_utils.weekday import Weekday
+from .constants import HTML_LINK, STATES, BUCKET
+from .. import (
+    get_structured_logger,
+)
+
+
+def split_reporting_schedule_dfs(input_df, flash_dir, lag):
+    """Separate the input df by reporting schedule (pre-determined).
+
+    Parameters
+    ----------
+    input_df: the df to split up
+    flash_dir: the string reference directory to find all files
+    lag: difference between reporting and reference date.
+
+    Returns
+    -------
+    df of streams updated daily, less often than daily BUT has enough data to use the AR method
+    and data that's updated very infrequently and there's not enough data to use the AR method.
+    """
+    rep_sched = pd.read_csv(f'{flash_dir}/reporting_sched_{lag}.csv', index_col=0)
+    min_cut = rep_sched.loc['min_cut'][0]
+    rep_sched = rep_sched.drop('min_cut')
+    glob_out_list = []
+    non_daily_ar = []
+    for i, df in rep_sched.groupby('0'):
+        fixed_sum = []
+        columns = []
+        for col in input_df.columns:
+            if col in df.index:
+                columns.append(col)
+                fixed_sum.append(input_df[col])
+
+        if len(fixed_sum) > 0:
+            fixed_sum_df = pd.concat(fixed_sum).to_frame().T
+            fixed_sum_df.columns = columns
+            fixed_sum_df.index = [input_df.index[0]]
+            if i ==1:
+                daily_df = fixed_sum_df
+            elif i >= min_cut:
+                glob_out_list.append(fixed_sum_df)
+            else:
+                non_daily_ar.append(fixed_sum_df)
+    return daily_df, pd.concat(non_daily_ar,axis=1) , pd.concat(glob_out_list, axis=1)
```

Comment:
Suggestion: return as tuple.

```suggestion
    return (daily_df, pd.concat(non_daily_ar,axis=1) , pd.concat(glob_out_list, axis=1))
```